pkg/ccl/backupccl/backupresolver/targets.go

```diff
@@ -437,6 +439,9 @@ func DescriptorsMatchingTargets(
 			}
 
 		case *tree.AllTablesSelector:
+			scName := p.Schema()
+			hasSchemaScope := p.ExplicitSchema && p.ExplicitCatalog
```

Comment:
can we add a comment:

If the target is fully qualified all tables selector, we only want to backup all tables in the specified schema.

Reply:
Done.